doc/_dev-guide/chapter_4/section_2/README.md

```diff
@@ -185,40 +364,190 @@ export CXX=clang++
+ Build options:
+  Generate manual pages       : no
+  Install manual pages        : no
+ Modules:
+  Module search path          : ${exec_prefix}/lib/syslog-ng
+  Sun STREAMS support (module): no
+  Darwin OSL support (module) : yes
+  OpenBSD syslog (module)     : no
+  SQL support (module)        : yes
+  PACCT module (EXPERIMENTAL) : no
+  MongoDB destination (module): yes
+  JSON support (module)       : yes
+  SMTP support (module)       : yes
+  HTTP support (module)       : yes
+  AMQP destination (module)   : yes
+  STOMP destination (module)  : yes
+  MQTT (module)               : yes
+  GRPC (module)               : yes
+  Cloud Auth (module)         : yes
+  GEOIP2 support (module)     : yes
+  Redis support (module)      : yes
+  Kafka support (module)      : yes
+  Riemann destination (module): yes, microseconds: no
+  Python                      : yes (pkg-config package: auto, packages: venv)
+  Python modules              : yes
+  Java                        : yes
+  Java modules                : yes
+  afsnmp module               : yes
+  eBPF module                 : no
+  native bindings             : yes
+  Example modules             : yes
+```
 
-For more details please see the [[actual state of supported features|dev-macos-mod-sup-status]], and the required [dependencies](#dependencies).
+The modules have the `:yes` status in the list will be compiled and installed. As you installed the required dependencies earlier there is no need to specify the modules you want to enable, the configuretion step will auto enable the modules that have all the dependencies, Of course, you can disable a given module directly if you do not need it (e.g. `--disable-kafka` will disable the kafka modules).
+
+For trying force enable a given module you can use e.g. `--enable-kafka` that will always try to search for all the module dependencies and stop the configuration if any of it is missing.
 
 #### Using cmake
 
-For the full feature set you can use
+If you have all the above mentioned dependencies installed, for the full (currently supported) feature set you can simply use
 
 ```shell
-cmake --install-prefix /full_path_of/installdir -B build . -Wno-dev -DIVYKIS_SOURCE=system -DENABLE_JAVA=OFF -DENABLE_JAVA_MODULES=OFF -DENABLE_PYTHON=ON -DENABLE_PYTHON_MODULES=ON -DBUILD_TESTING=OFF -DENABLE_AFSMTP=OFF -DENABLE_PACCT=OFF -DENABLE_CPP=ON -DENABLE_GRPC=ON --fresh
+# NOTE: you might want to use the --install-prefix /full_path_of_your/installdir/ parameters as well, see bellow Warning!
+cmake --install-prefix /full_path_of_your/installdir -B build . -Wno-dev -DIVYKIS_SOURCE=system --fresh
 ```
 
-### Compile and install
+**Warning:** By a good chance, you might want to install the self built instance first to a custom location to prevent overwriting a possibly already existing installed version. In that case pass `---install-prefix /full_path_of_your/installdir/` parameter to the `cmake` command like you see in the above steps.
```

Comment:
I do not understand this  "By a good chance" part, it might not make any sense.  You might want to reword this sentence-